test/browser_reporting.js

```diff
@@ -29,7 +32,15 @@ function maybeReportResultToServer(result) {
   }
 }
 
-function reportErrorToServer(message) {
+function reportStderrToServer(message) {
+  if (typeof ENVIRONMENT_IS_NODE !== 'undefined' && ENVIRONMENT_IS_NODE) {
+    err(message);
+  } else {
+    fetch(`${reportingURL}?stderr=${encodeURIComponent(message)}`);
+  }
+}
+
+function reportStdoutToServer(message) {
   if (typeof ENVIRONMENT_IS_NODE !== 'undefined' && ENVIRONMENT_IS_NODE) {
     err(message);
   } else {
```

Comment:
The line right beneath this diff says "stderr" but I think it should now say "stdout"?

Reply:
Oops, good catcht.  Fixed.